### ISSUE TITLE
[opencog] CogServer module paths are not correct

### ISSUE THREAD

**Issue by Radivarig** [opencog/opencog-nix#30]
Moving here from #28

**Comment by Radivarig** [comment]
Adding default paths in `CogServer.cc`:
```diff
+        module_paths.push_back("../../opencog/cogserver/server/");
+        module_paths.push_back("../../opencog/cogserver/shell/");
```
seems to work for ShellUTest.

**Comment by linas** [comment]
That will find the modules, when they are in the build-tree. But on non-nix systems, that is not the install location.

what's $CWD?  is it enough to say
```
module_paths.push_back("./server/");
module_paths.push_back("./shell/");
```

**Comment by Radivarig** [comment]
I figured now that these are not needed when `LD_LIBRARY_PATH` is extended with location of *.so files that are produced by the build. As I did in `postBuild` phase for [atomspace here](https://github.com/opencog/opencog-nix/blob/a53bf1d85881584078ff9bc6496102625b5f027e/packages/atomspace.nix#L70), I added it here as well for some later tests to work. So, changing `CogServer.cc` is not necessary.

cwd was: `build/tests/server` without any *.so files.
